docs/installation.rst

```diff
@@ -357,6 +357,19 @@ activating your virtual environment::
 
     pip install -U borgbackup  # or ... borgbackup[llfuse/pyfuse3]
 
+When doing manual pip installation, man pages are not automatically
+installed (this is likely handled by your package manager). You can run
```

Comment:
the text in parens is rather confusing.

guess it should be either left away or be something like:
```
(unlike when borg is installed via a linux distribution's package manager)
```